.github/workflows/on-pull-request-opened-and-synchronize.yml

```
@@ -166,6 +169,12 @@ jobs:
         # below, before merging.
         continue-on-error: true
         run: pnpm exec playwright test --project visuals --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+      - name: Set failed
```

What to name this job? 🤔

---

I'm thinking maybe something with "output" since it's used as `needs.test-visuals.outputs.failed` below.

```suggestion
      - name: Set failed output
```

```suggestion
      - name: Set output failed flag
```

More verbose, but we only set the flag if the outcome is failed.

```suggestion
      - name: Set output state on failure
```

But I'm also good with what you have here.